packages/account/src/Components/form-field-info/form-field-info.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Popover } from '@deriv/components';
+import { TPopoverProps } from '@deriv/components/src/components/types';
+import { isMobile } from '@deriv/shared';
+
```

Comment:
why do we need all these comments as code looks like self descriptive?

Reply:
This seems reasonable to put documentation over components